### PR TITLE
Mehron's files

### PR DESCRIPTION
**Pull request by openly-retro**
Just some content with a quote that I like.

### CODE REVIEW

**Comment by nw0428** [comment]
Great start!
Definitely want to remove the nmml_about_me folder AND add a link both from the index page to your page and vice versa.
Awesome quote! =)

File: mehron.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    <p>Our posturings, our imagined self-importance, the delusion that we have some privileged position in the Universe, are challenged by this point of pale light. Our planet is a lonely speck in the great enveloping cosmic dark. In our obscurity, in all this vastness, there is no hint that help will come from elsewhere to save us from ourselves.</p>
+
+    <p>The Earth is the only world known so far to harbor life. There is nowhere else, at least in the near future, to which our species could migrate. Visit, yes. Settle, not yet. Like it or not, for the moment the Earth is where we make our stand.</p>
+
+    <p>It has been said that astronomy is a humbling and character-building experience. There is perhaps no better demonstration of the folly of human conceits than this distant image of our tiny world. To me, it underscores our responsibility to deal more kindly with one another, and to preserve and cherish the pale blue dot, the only home we've ever known.</p>
+    <p>&emdash; Carl Sagan</p>
```
Comment (nw0428) [inline mehron.html]: `&emdash` does not get rendered correctly on the page.

**Comment by nickemmons** [comment]
That's an awesome quote!

**Comment by MicahMartin** [comment]
I second what Nick said. Carl Sagan is the f***ing man

lookgoodtome

**Comment by nw0428** [comment]
LGTM

**Pull request merged**